project/Dependencies.scala

```diff
@@ -55,8 +55,8 @@ object Dependencies {
       val cache       = "com.typesafe.play" %% "play-cache"        % version
       val test        = "com.typesafe.play" %% "play-test"         % version
       val jdbc        = "com.typesafe.play" %% "play-jdbc"         % version
-      val json        = "com.typesafe.play" %% "play-json"         % "2.6.14"
-      val jsonJoda    = "com.typesafe.play" %% "play-json-joda"    % "2.6.14"
+      val json        = "com.typesafe.play" %% "play-json"         % "2.9.1"
+      val jsonJoda    = "com.typesafe.play" %% "play-json-joda"    % "2.9.1"
       val mailer      = "com.typesafe.play" %% "play-mailer"       % "6.0.1"
       val mailerGuice = "com.typesafe.play" %% "play-mailer-guice" % "6.0.1"
```

Comment:
we have already gone through that exercise, and an upgrade of Play needs to come with its companion libraries like the mailer here or Silhouette...

Reply:
what excercise ?
i think mailer is straightforward but when i attempted to upgrade sillhouette alot of things broke

Reply:
my original attempt was to upgrade everything but it led to alot of errors